src/components/Sum.jsx

```diff
@@ -1 +1,20 @@
-// Don't forget to add default values an check types of props
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = (a = 0, b = 0) (
+    <p>
+        {'Sum of'}
+        {a} 
+        {'and'}
+        {b} 
+        {'is'}
+        {a + b}
+    </p>
```

Comment:
```suggestion
    <p>
      {'Sum of'}
      {a} 
      {'and'}
      {b} 
      {'is'}
      {a + b}
    </p>
```
Двух отступов достаточно